chapters/synchronous.tex

```diff
@@ -720,11 +720,13 @@ \subsection{Sub-Clock Conversion Operators}\label{sub-clock-conversion-operators
 Clock u  = Clock(3, 10);            // ticks: 0, 3/10, 6/10, $\ldots$
 Clock y1 = shiftSample(u, 1, 3);    // ticks: 1/10, 4/10, $\ldots$
 // Event clock
-Clock u = Clock(sin(2 * pi * time) > 0, startInterval = 0.0);
+Integer revolutions=integer(time);
+Clock u = Clock(change(revolutions), startInterval = 0.0);
                                     // ticks: 0.0, 1.0, 2.0, 3.0, $\ldots$
 Clock y1 = shiftSample(u, 2);       // ticks: 2.0, 3.0, $\ldots$
 Clock y2 = shiftSample(u, 2, 3);    // error (resolution must be 1)
 \end{lstlisting}
+Note: The events generated by \lstinline!change(revolutions)! are at the same time as \lstinline!sin(2 * pi * time) > 0!, but it is numerically more robust.
```

Comment:
I don't see the need to include this note now that both occurrences of `sin(2 * pi * time) > 0` have been removed.
```suggestion
```

Reply:
I agree. I think it is confusing to mention `sin(2 * pi * time) > 0` when it does not occur anywhere now.